docs/gitbook/development-guide/ChangeDependencies.md

```diff
@@ -0,0 +1,66 @@
+# Add or Update Dependencies 
+
+To know about the required dependencies, their minimum required versions check the requirements*.txt files.
+Whenever we pin to a specific version in requirements*.in, we add a comment explaining why we are doing so. 
+We also have comments on their use.
+
+## To install or update all dependencies
+
+**Note** : The pip-compile command lets you compile a requirements*.txt from your dependencies, specified in either setup.py or requirements.in.
```

Comment:
Can you also add a link to [pip-tools](https://github.com/jazzband/pip-tools) and mention that it installs the `pip-compile` command?